src/cmd-remote-build-container

```diff
@@ -124,6 +125,17 @@ def main():
         commit = runcmd(cmd, quiet=True, capture_output=True).stdout.strip().decode()
         shortcommit = commit[0:7]
         logging.info(f"Translated {args.git_url}#{args.git_ref} into {shortcommit}")
+        # Add some information about the commit to labels for the container
```

Comment:
There is a standard schema for this:
https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys

We're using that schema in the base image: https://github.com/coreos/coreos-assembler/blob/1f62f5ad9c89ff84acf67ce17bd934f01bea8075/src/cmd-build#L419

Reply:
are `org.opencontainers.image.source` and `org.opencontainers.image.revision` sufficient here? I'll just use that if so.

Reply:
My opinion is: yes.  Those two fields are the minimal things necessary.  I don't see a strong motivation to duplicate things like the authorship etc.  The source of truth for that data is git.

Reply:
Alternatively, anything we think should/must be there is also something we should be injecting into the base image build.  I'm arguing to keep them consistent.

Reply:
ok I updated it.